gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -262,13 +262,30 @@ case object RandomUUID extends ElPart[UUID] {
   def apply(session: Session): Validation[UUID] = version4UUID().success
 }
 
+case object RandomInt extends ElPart[Int] {
+  def apply(session: Session): Validation[Int] = ThreadLocalRandom.current().nextInt().success
+}
+
+final case class RandomIntRange(min: Int, max: Int) extends ElPart[Int] {
+  def apply(session: Session): Validation[Int] = (min + ThreadLocalRandom.current().nextInt((max - min) + 1)).success
```

Comment:
Please:
* add some validation on the input parameters, that `min < max`, using `require`
* compute `max - min + 1` only once (private attribute, no need for the extra parens)

`val length = {
  require(max > min, "some error message")
  max - min + 1
}`

Reply:
@slandelle  Honestly, I thought about that, but I didn't find any other `require` there. I saw only some failure messages in `ElMessages` which related more to failures in EL logic, function, etc. And I decided maybe it's preferable to "trust" in exceptions which can be thrown by.

Reply:
When possible, it's better to have some validation early when the String is parsed than having some Exceptions thrown when the Simulation is running.
If we spot other places where such requirements would make sense, let's add that in a follow-up PR.